README.md

```diff
@@ -464,9 +464,6 @@ You can find an example of a configuration file [here](https://github.com/maplib
 # Database connection string
 connection_string: 'postgres://postgres@localhost:5432/db'
 
-# Trust invalid certificates. This introduces significant vulnerabilities, and should only be used as a last resort.
-danger_accept_invalid_certs: false
```

Comment:
Why remove `danger_accept_invalid_certs`?

Reply:
i guess we can keep it in the readme - as long as it is not in the test (which fails now because this value is unrecognized when compiling without SSL... which might actually be a bad thing - we should probably build WITH ssl when testing in CI)